WikipediaUnitTests/CLLocationManagerMock.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+/// Mock for `CLLocationManager`
+final class CLLocationManagerMock: CLLocationManager {
+
+    // MARK: - Override CLLocationManagerProtocol
+
+    var _heading: CLHeading?
+    override var heading: CLHeading? {
+        return _heading
+    }
+
+    var _location: CLLocation?
+    override var location: CLLocation? {
+        return _location
+    }
+
+    override func requestAlwaysAuthorization() { }
+
+    override func requestWhenInUseAuthorization() { }
+
+    override func startMonitoringSignificantLocationChanges() { }
+
+    override func stopMonitoringSignificantLocationChanges() { }
+
+    override func startUpdatingLocation() {
+        updatingLocation = true
+    }
+
+    override func stopUpdatingLocation() {
+        updatingLocation = false
+    }
+
+    override func startUpdatingHeading() {
+        updatingHeading = true
+    }
+
+    override func stopUpdatingHeading() {
+        updatingHeading = false
+    }
+
+    static var _authorizationStatus: CLAuthorizationStatus = .authorizedAlways
+    override class func authorizationStatus() -> CLAuthorizationStatus {
+        return _authorizationStatus
+    }
+
+    // MARK: - Test properties
+
+    var updatingLocation: Bool?
+    var updatingHeading: Bool?
+
+    static func resetMockData() {
```

Comment:
It would be nice to have a comment here explaining what this method does.

Reply:
`Reset CLLocationManager to the default values:
- `